audformat/core/utils.py

```diff
@@ -1074,3 +1074,232 @@ def union(
     index = index.drop_duplicates()
 
     return index
+
+
+def explode_overlapping_segments(
+        obj: typing.Union[pd.Series, pd.DataFrame]
+) -> pd.Series:
+    r"""Explodes overlapping segments.
+
+        This method assumes that labels are one hot encoded vectors in form
+        of tuples e.g (1, 0, 0) or lists [1, 0, 0].
+
+        The labels in the returned object will be one hot encoded in a tuple,
+        whether they were initially in a tuple or a list.
+
+        Args:
+            obj: Pandas Serie or Dataframe that index's conform to
+            :ref:`table specifications <data-tables:Tables>`.
+
+        Returns:
+            Pandas Dataframe with overlapping segments exploded.
+
+        Raises:
+            ValueError: if the dataframe columns aren't packed into a single
+            column.
+            ValueError: If the labels are single classes and not one-hot
+            encoded vectors
+
+        Example:
+            >>> obj = pd.Series(index=pd.MultiIndex.from_arrays([
+            ...     ['audio_file.wav',
+            ...     'audio_file.wav'],
+            ...     [pd.to_timedelta('0 days 00:00:00.861768592'),
+            ...     pd.to_timedelta('0 days 00:00:03.428162853')],
+            ...     [pd.to_timedelta('0 days 00:00:09.581768592'),
+            ...     pd.to_timedelta('0 days 00:00:04.488162853')]
+            ...     ], names=('file', 'start', 'end')), data=[(1, 0, 0, 0, 0)
+            ...     , (0, 0, 1, 0, 0)])
+            ...     )
+            file            start                      end
+            audio_file.wav  0 days 00:00:00.861768592  0 days 00:00:09.581768592    (1, 0, 0, 0, 0)  # noqa
+                            0 days 00:00:03.428162853  0 days 00:00:04.488162853    (0, 0, 1, 0, 0)  # noqa
+            >>> obj = explode_overlapping_segments(obj)
+            file            start                      end
+            audio_file.wav  0 days 00:00:00.861768592 0 days 00:00:03.428162853  (1, 0, 0, 0, 0)  # noqa
+                            0 days 00:00:03.428162853 0 days 00:00:04.488162853  (1, 0, 1, 0, 0)  # noqa
+                            0 days 00:00:04.488162853 0 days 00:00:09.581768592  (1, 0, 0, 0, 0)  # noqa
+
+    """
```

Comment:
You can specify the `noqa` at the end of the docstring:

```suggestion
            file            start                      end
            audio_file.wav  0 days 00:00:00.861768592  0 days 00:00:09.581768592    (1, 0, 0, 0, 0)
                            0 days 00:00:03.428162853  0 days 00:00:04.488162853    (0, 0, 1, 0, 0)
            >>> obj = explode_overlapping_segments(obj)
            file            start                      end
            audio_file.wav  0 days 00:00:00.861768592 0 days 00:00:03.428162853  (1, 0, 0, 0, 0)
                            0 days 00:00:03.428162853 0 days 00:00:04.488162853  (1, 0, 1, 0, 0)
                            0 days 00:00:04.488162853 0 days 00:00:09.581768592  (1, 0, 0, 0, 0)

    """  noqa: E501
```